docs/understand/programming_model_reference.rst

```diff
@@ -87,9 +87,9 @@ of. It relaxes some restrictions of the :ref:`inherent_thread_model`
 imposed by the strict 1:1 mapping of architectural details to the programming
 model.
 
-The rich set of APIs introduced by Cooperative Groups allow the programmer to 
-define their own set of thread groups which may fit their user-cases better than 
-those defined by the hardware. The set of implicit groups by kernel launch 
+The rich set of APIs introduced by Cooperative Groups allow the programmer to
```

Comment:
Could we add a reference to the Cooperative Groups API docs here? Or another topic that adds details?